client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -48,6 +48,7 @@ const ConnectedSingleSiteJetpack = connectOptions( ( props ) => {
 	} = props;
 
 	const displayUpsellBanner = isAtomic && ! requestingSitePlans && currentPlan;
+	const upsellUrl = isAtomic && `/plans/${ siteId }?feature=upload-themes&plan=business-bundle`;
```

Comment:
Nitpick: It's preferable to use the constants defined in the `calypso-products` package:

https://github.com/Automattic/wp-calypso/blob/92641abe2f288072e1904cce370d743e463be5a8/packages/calypso-products/src/constants/features.ts#L61

https://github.com/Automattic/wp-calypso/blob/92641abe2f288072e1904cce370d743e463be5a8/packages/calypso-products/src/constants/wpcom.ts#L13